docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,367 @@
+* EAP-Type: The EAP type defines the authentication method supported and should be set to EAP-TLS to integrate with the PacketFence PKI.
+* Security type: The security type should be set to WPA2 to integrate with the PacketFence PKI.
+* PKI Provider: This should match the provider you configured earlier in the section on providers.
+
+The following is an example on how to configure an EAP-TLS connection for Windows/Android/OS X/iOS
+
+image::docs/images/scep-ms-pki-eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+
+OS X/iOS require signing the provisioning profile with a Certification Authority already trusted by the device such as e.g. VeriSign.
+Configuring this has to be done in the 'Signing' tab in the "Apple devices". 
+
+image::docs/images/packetfence-pki-eaptls-sign-example.png[scaledwidth="100%",alt="Signing provisioner"]
+
+Fill out the fields with the contents of the Base64 encoded certificates.
+To extract this information from a pem formatted certificate, copy the file content included between the begin and end tag, not including the delimiters themselves.
+
+Certificate file example:
+
+----
+----- BEGIN CERTIFICATE -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END CERTIFICATE -----
+----
+
+Copy everything between the BEGIN and END lines, but not the lines themselves.
+Repeat this operation for the certificate key and intermediate certificate if any.
+
+Private key file example:
+
+----
+----- BEGIN PRIVATE KEY -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END PRIVATE KEY -----
+----
+
+
+Portal Profiles Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provisioners have to be enabled on the Portal Profiles configuration in the PacketFence GUI.
+
+Under Configuration > Main > Portal Profiles, select each of the provisioners created above which should be active for the profile.
+If no portal profile is defined, configure the "default" profile to use the provisioners created.
+
+NOTE: If you use two different portal profiles for the open and secure networks, make sure you configure the provisioners on both profiles.
+
+
+Passthroughs required for Android
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Android devices require passthroughs to be created to allow them to fetch the configuration application from the Play Store.
+
+Add the following to the "Trapping" section of the Configuration tab in the PacketFence GUI.
+
+----
+passthrough=enabled
+passthroughs=*.ggpht.com,*.googleusercontent.com,android.clients.google.com,
+  *.googleapis.com,*.android.clients.google.com,*.gvt1.com
+----
+
+Debugging MSPKI integration with PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This a way to do the procedure of enrollment manually, mainly use for debugging purpose.
+
+First you need to generate a request and its private key via mkrequest. Go in the folder were sscep is installed, it should be /usr/local/scep
```

Comment:
The folder path will most likely change, depending on where we are installing the rpm of scep.